networking/network_security/AdminNetworkPolicy/ovn-k-anp-banp-metrics.adoc

```diff
@@ -0,0 +1,11 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ovn-k-anp-banp-metrics"]
+= Monitoring and managing ANP and BANP
+include::_attributes/common-attributes.adoc[]
+:context: ovn-k-anp-banp-metrics
+
+toc::[]
+
+`AdminNetworkPolicy` and `BaselineAdminNetworkPolicy` have many metrics for monitoring and managing your policies. Follow the instructions, in this section to help you with your policy maintenance.
```

Comment:
```suggestion
`AdminNetworkPolicy` and `BaselineAdminNetworkPolicy` have metrics that can be used for monitoring and managing your policies. See the below table for more details
```
what are the instructions? :P

Reply:
Yeah, that needs to go. I didn't remove it with the removal of the `oc describe anp`. Doing that now.